app/views/inventory_foods/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="inventory_form">
+    <%= form_with model: @InventoryFood, url:inventory_foods_path do |form| %>
+        <%= form.select :food_id, @foods.map {|food| [food.name, food.id] }, {:include_blank => "-", :required => true} %>
+        <%= form.label :food_id, "Food"%>
+
+        <%= form.select :inventory_id, @inventory.map {|invent| [invent.name, invent.id]}, {:include_blank => "-", :required => true} %>
+        <%= form.label :inventory_id, "Inventory" %>
+
+        <%= form.number_field :quantity, required: true %>
+        <%= form.label :quantity, "Quantity" %>
+
+        <%= form.submit "Add Ingredient"%>
+    <% end %>
+</div>
```

Comment:
This block is not needed. Inventory foods is not our task and on UI there is no link to this file.